azkaban-exec-server/src/main/bash/start-exec.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
-
 script_dir=$(dirname $0)
 
+source "${script_dir}/internal/startup-shared.sh"
+
 # pass along command line arguments to the internal launch script.
-${script_dir}/internal/internal-start-executor.sh "$@" >executorServerLog__`date +%F+%T`.out 2>&1 &
+log_file="executorServerLog__$(date +%F+%T).out"
+startup_cmd="${script_dir}/internal/internal-start-executor.sh $@"
```

Comment:
I like the way you break up long command lines into smaller more readable pieces.

Reply:
This is not just for readability but to make it clear that these are the 2 arguments passed to that function. These two are quoted later for that reason.

Reply:
I see.